narwhal/storage/src/consensus_store.rs

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::NodeStorage;
+use config::AuthorityIdentifier;
+use std::collections::HashMap;
+use store::rocks::{open_cf, DBMap, MetricConf, ReadWriteOptions};
+use store::{reopen, Map, TypedStoreError};
+use types::{
+    CommittedSubDag, CommittedSubDagShell, CompressedCommittedSubDag, CompressedCommittedSubDagV2,
+    Round, SequenceNumber, StoreResult,
+};
+
+/// The persistent storage of the sequencer.
+pub struct ConsensusStore {
```

Comment:
I've moved this out of the `narwhal-types` crate so can be where the rest of the stores are.